fv3core/utils/global_constants.py

```diff
@@ -35,3 +35,10 @@
 t_sub = 184.0  # min temp for sublimation of cloud ice
 DC_ICE = C_LIQ - C_ICE
 LI0 = HLF - DC_ICE * TICE
+
+# grid constants
+LON_OR_LAT_DIM = "lon_or_lat"
+TILE_DIM = "tile"
+CARTESIAN_DIM = "xyz_direction"
+N_TILES = 6
+RIGHT_HAND_GRID = False
```

Comment:
Because these are only used within `fv3core.grid`, could you put them somewhere within that namespace to avoid these getting used outside of the grid code? The (very good) way the grid code is written, you are allowed to use an entirely different name to refer to these dimensions in a different part of the code.

If you only import these in one file each, you can put them in that file.

Reply:
Flagging here as it's probably a bit out-of-scope for this PR, but we should maybe put some TODOs in around this issue